ajax/libs/jqcloud/package.json

```diff
@@ -11,5 +11,17 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/lucaong/jQCloud"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/lucaong/jQCloud.git",
+    "fileMap": [
+      {
+        "basePath": "jqcloud",
+        "files": [
+          "**/*"
```

Comment:
@maruilian11 Because Peter hope me to finish this PR ASAP. Namely, without upstream update (change naming scheme) and manually add new versions or change the filename.